### DISCUSSION TITLE
get_hostname not working in kong 2.7.0

### DISCUSSION THREAD

**Discussion by kollol-chowdhury-incontact** [Kong/kong#8281] [Help] [answered]
In Kong 2.0.3,there was a function in kong utils called get_hostname.
When I upgraded to kong 2.7.0,the plugin which uses get_hostname is failing as 2.7.0 utils part doesn't seem to have get_hostname defined.

Has there been any change?how do we use that functionality in kong 2.7.0.

TIA

**Comment by kollol-chowdhury-incontact** [comment]
Hi @kikito ,could you please help me in this.

TIA

**Comment by kikito** [comment, marked as answer]
Hello, please try using this PDK function instead: [kong.node.get_hostname](https://docs.konghq.com/gateway/2.7.x/pdk/kong.node/#kongnodeget_hostname)
